### PR TITLE
Update ScrollViewer.md

### PR DESCRIPTION
**Pull request by Wingnutt**

### CODE REVIEW

File: content/How_To/GUI/ScrollViewer.md [hunk 1]
```diff
@@ -72,6 +72,12 @@ myTextBlock.resizeToFit = true;
 
 * [Playground Example - Scroll Viewer with Adjusting TextBlock](https://www.babylonjs-playground.com/#3EF49E#5)
 
+## Live-Updating and Multiple TextBlocks
+
+The ScrollViewer accepts only ONE textblock child. You can modify its _.text_ property, including \\n linebreaks, to add/remove text content, using that single textBlock's font styles.
```
Comment (deltakosh) [inline content/How_To/GUI/ScrollViewer.md]: we should mention that any container or control can be used as well

Comment (deltakosh) [reply content/How_To/GUI/ScrollViewer.md]: ping :)

Comment (Wingnutt) [reply content/How_To/GUI/ScrollViewer.md]: Oh, sorry.  Yeah, adjust as wanted.  I thought I DID mention it in the 2nd paragraph.  Probably have it worded incorrectly.

"Scrollviewer can contain a SINGLE control of ANY type, including containers of one or many other controls.  Some controls may need a _height_ setting." 

?

Sorry for my slow reply, here.  I'm not sure how to word it.  Please run with it, in any way you think is correct.  I'm cool... and still learning.  :)

Comment (deltakosh) [reply content/How_To/GUI/ScrollViewer.md]: Yeah something like that. The only constraint is that height cannot be a percentage

Comment (Wingnutt) [reply content/How_To/GUI/ScrollViewer.md]: Yeah... good point.  :)

"Scrollviewer can contain a SINGLE control of ANY type, including containers of one or many other controls. Some controls may need a non-percentage height."

I dunno, DK.  Do it however you wish.  Whatever it takes so I can "git" out of github and back to the forum where I belong.  heh.

https://github.com/Wingnutt/Documentation/blob/patch-1/content/How_To/GUI/ScrollViewer.md#live-updating-and-child-containers

Another version.  :)  I probably said something wobbly, again, but I hope not.  Do I need to re-PR or something weird?  Where am I?  Let me out of here.  heh.

Just use your magic, DK, and force this to however you think it should be, please.  :)

Comment (deltakosh) [reply content/How_To/GUI/ScrollViewer.md]: ok second version is great

**Review by deltakosh — APPROVED** [review]

**Pull request merged**